.gitignore

```diff
@@ -1,3 +1,4 @@
 *.bin
 .hypothesis
 dist/
+__pycache__
```

Comment:
Any reason to remove `__pycache__`? We probably don't want this in the repo

Reply:
maybe there's something i can tweak to avoid the `__pycache__` folder from being generated in my local repo. however i think it's fairly common that this happens and we most likely don't want the pycache folder to be commited, or is there something i'm missing?

Reply:
Oh sorry, I thought this was a removal not an addition! Ignore me